vaadin-grid-flow-demo/src/main/java/com/vaadin/flow/component/treegrid/demo/TreeGridDemo.java

```diff
@@ -146,4 +149,78 @@ public boolean hasChildren(Account item) {
 
         addCard("TreeGrid with lazy loading", grid);
     }
+
+    // Drag and drop
+    private Department draggedItem;
+
+    private void createDragAndDrop() {
+        DepartmentData departmentData = new DepartmentData();
+        // begin-source-example
+        // source-example-heading: Drag and drop
+        TreeGrid<Department> treeGrid = new TreeGrid<>();
+
+        treeGrid.setDataProvider(
+                new AbstractHierarchicalDataProvider<Department, Object>() {
```

Comment:
Is this code necessary for this demo? Consider abstracting it out to a class or method.

Reply:
Done.